--- public/javascripts/node.js.orig	2014-02-12 02:10:44.000000000 +0000
+++ public/javascripts/node.js	2014-02-12 02:26:12.000000000 +0000
@@ -49,7 +49,7 @@
           } else {
             for (_i = 0, _len = data.length; _i < _len; _i++) {
               node = data[_i];
-              node.set('isSelected', true);
+              node.set('isSelected', false);
             }
             return set.add.apply(set, data);
           }
